### DISCUSSION TITLE
icon request for gun cash

### DISCUSSION THREAD

**Discussion by sschemer** [elrumo/macOS_Big_Sur_icons_replacements#1377] [Icon request] [answered]
GnuCash is personal and small-business financial-accounting software, freely licensed under the [GNU](https://www.gnu.org/) GPL.

website: [https://www.gnucash.org/](https://www.gnucash.org/)

official logo: 

 ![gun cash logo](https://www.gnucash.org/images/gnc-download.png)

Thanks in advance.

**Comment by ThaliaMyers** [comment, marked as answer]
Done. It is on my profile page waiting for approval.
https://macosicons.com/#/u/CaedmonM

**Reply by sschemer** [reply]
Thanks!

**Reply by ThaliaMyers** [reply]
You're Welcome!
